pytorch/pytorch_edgeml/graph/rnn.py

```diff
@@ -1050,29 +1049,42 @@ def forward(self, input, hiddenState=None, cellState=None):
 
 class SRNN2(nn.Module):
 
-    def __init__(self, inputDim, outputDim, hiddenDim0, hiddenDim1, cellType):
+    def __init__(self, inputDim, outputDim, hiddenDim0, hiddenDim1, cellType, dropoutProbability, **cellArgs):
```

Comment:
1. break lines.
2. We should have separate dropout number to each layer. Also we should have a default argument for the dropout for people who don't want to use it.
```
def __init__(self, inputDim, outputDim, hiddenDim0, hiddenDim1, cellType,
                   dropout0=1.0, dropout1=1.0, **cellArgs)
```

Even though having a dropout layer with 1.0 dropout (ie. no dropout) might be functionally same as having no dropout, there might be performance hits because of this additional layer. I guess you could default to `dropout0=None` and then not add the dropout layer if `dropout0` remains none inside `__init__`. 

Also, please `assert 0< dropout0 <= 1.0`